### ISSUE TITLE
Remember conversation

### ISSUE THREAD

**Issue by mindriddler** [mindriddler/discord-bot#17]
Right now the bot cannot remember conversations
There was ways to fix this, will have to look into it

One solution is to store the history of a just in a python dict.. but i think that will be messy and also kinda bad.

The better solution would be to store the data in a small database, 

* One database
* A table for each user
	* The tables will be created dynamicly for each user
	* Maybe add a way to remove one tables if they been inactive for like 1 week?

Postgres? SQLite? MySQL? 
https://medium.com/@avra42/how-to-build-a-chatbot-with-chatgpt-api-and-a-conversational-memory-in-python-8d856cda4542

**Comment by PapaPeskwo** [comment]
Pinecone can solve this problem and provides an API
https://www.pinecone.io/

**Comment by mindriddler** [comment]
> Pinecone can solve this problem and provides an API
> https://www.pinecone.io/

Yeah, but since the free tier is gated behind a waitlist and its quite expensive.. $70 per month as the cheapest option, which is way to much.

Also, I think it would be better to not be dependant on more things then what the bot already is (OpenAI ans Discord) 

Its an option, but I think I will go with another option.
